champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/axe/Leap.java

```diff
@@ -79,6 +80,7 @@ public void doLeap(Player player, boolean wallkick) {
 
         player.getWorld().spawnEntity(player.getLocation(), EntityType.LLAMA_SPIT);
         player.getWorld().playSound(player.getLocation(), Sound.ENTITY_BAT_TAKEOFF, 2.0F, 1.2F);
+        championsManager.getEffects().addEffect(player, EffectType.NOFALL, 2000L);
```

Comment:
?

Reply:
so that leap doesn't deal fall damage to the user every time you use it

Reply:
On mineplex it seems they just reduced fall damage by 2 if you happened to have leap equipped

`	@EventHandler(priority = EventPriority.HIGH)
	public void Damage(CustomDamageEvent event)
	{
		if (event.IsCancelled())
			return;

		if (event.GetCause() != DamageCause.FALL)
			return;

		Player player = event.GetDamageePlayer();
		if (player == null)		return;

		int level = getLevel(player);
		if (level == 0)			return;

		event.AddMod(null, GetName(), -2, false);
	}`

Reply:
Honestly Id much prefer to just negate all damage within a certain time much like siesmic slam, if its too long a duration you can reduce it but it seems much more fitting and less annoying(nobody used leap on mineplex)

Reply:
that isn't why nobody used it, all this does is create inconsistent results if it takes you longer than 2 seconds to hit the ground

Reply:
I agree it could create inconsistent results though leap needs something to prevent it from taking fall damage while using the ability and I dont think net reducing an assassins fall damage with the ability equipped is the way to go

Reply:
fixed